### PR TITLE
WET-347

### PR DESCRIPTION
**Pull request by ouafaaetta**

### CODE REVIEW

File: components/header-rwd/index.json-ld [hunk 1]
```diff
@@ -7,14 +7,14 @@
 		"modified": "dct:modified"
 	},
 	"title": {
-		"en": "Well header responsive",
-		"fr": "Entête well responsive"
+		"en": "Context-specific feature",
+		"fr": "Liste - Style additionel"
```
Comment (duboisp) [inline components/header-rwd/index.json-ld:11]: Ceci ne correspond pas au composant "well header responsive"

**Review by duboisp — CHANGES_REQUESTED** [review]

File: components/header-rwd/index.json-ld (hunk 1 above)
Comment (ouafaaetta) [reply components/header-rwd/index.json-ld]: done

**Comment by duboisp** [comment]
There is a merge conflict that need to be resolved

**Review by duboisp — DISMISSED** [review]
Reviewed, the change looks good.

I will only update the _data/component.json generated file into this PR, so the change will be immediately effective.

**Review by duboisp — APPROVED** [review]

**Pull request merged**